tests/utils/test_session_token.py

```diff
@@ -80,3 +83,22 @@ async def test_invalid_token_parameters(token, parameter, value, error):
     header = generate_auth_header(token)
     with pytest.raises(error):
         SessionToken.from_header(header, API_KEY, API_SECRET)
+
+
+def parse_session_token(request: Request):
+    SessionToken.from_header(request.headers.get('Authorization'), API_KEY, API_SECRET)
+
+
+@pytest.mark.asyncio
+async def test_depends(token):
+    app = FastAPI()
+
+    @app.get('/token')
+    async def token_endpoint(token: SessionToken = Depends(parse_session_token)):
+        return token
+
+    client = TestClient(app=app)
+
+    header = generate_auth_header(token)
+
+    client.get('/token', headers={'Authorization': header})
```

Comment:
What is the actual test? Is it just checking that the code is running without exception?
I guess that's enough

Reply:
I agree, it was more of a sanity check to show that the dependency works (they're quite touchy).